app/src/main/java/com/codepath/chefster/activities/DishListActivity.java

```diff
@@ -0,0 +1,41 @@
+package com.codepath.chefster.activities;
+
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+
+import com.codepath.chefster.R;
+import com.codepath.chefster.fragments.FavoritesFragment;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+public class DishListActivity extends AppCompatActivity implements FavoritesFragment.OnFavoritesInteractionListener {
+
+    @BindView(R.id.toolbar) Toolbar toolbar;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_dish_list);
+
+        ButterKnife.bind(DishListActivity.this);
+
+        setSupportActionBar(toolbar);
+
+        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
```

Comment:
can you check for:
`if (getSupportActionBar() != null)`
Otherwise we could have a null pointer exception